openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config

```diff
@@ -1,4 +1,5 @@
 API_NODEJS_BUILD_NAME=eagle-api
+PI_MINIO_BUILD_NAME=eagle-api-minio
```

Comment:
Minor copy/paste, looks like the first char got clipped out

Reply:
Done